combat.sk

```diff
@@ -374,7 +330,186 @@ trails and tales update:
+		(amethyst [shard]|purple) rib = - {minecraft:trim={"material":"minecraft:amethyst","pattern":"minecraft:rib"}}
+		([nether] quartz|white) rib = - {minecraft:trim={"material":"minecraft:quartz","pattern":"minecraft:rib"}}
+		(netherite|black) rib = - {minecraft:trim={"material":"minecraft:netherite","pattern":"minecraft:rib"}}
+		(diamond|light blue) rib = - {minecraft:trim={"material":"minecraft:diamond","pattern":"minecraft:rib"}}
+		(gold|yellow) rib = - {minecraft:trim={"material":"minecraft:gold","pattern":"minecraft:rib"}}
+		(iron|grey|gray) rib = - {minecraft:trim={"material":"minecraft:iron","pattern":"minecraft:rib"}}
+		(copper|brown) rib = - {minecraft:trim={"material":"minecraft:copper","pattern":"minecraft:rib"}}
+
+		(emerald|green) eye = - {minecraft:trim={"material":"minecraft:emerald","pattern":"minecraft:eye"}}
+		(redstone|red) eye = - {minecraft:trim={"material":"minecraft:redstone","pattern":"minecraft:eye"}}
+		(lapis [lazuli]|blue) eye = - {minecraft:trim={"material":"minecraft:lapis","pattern":"minecraft:eye"}}
+		(amethyst [shard]|purple) eye = - {minecraft:trim={"material":"minecraft:amethyst","pattern":"minecraft:eye"}}
+		([nether] quartz|white) eye = - {minecraft:trim={"material":"minecraft:quartz","pattern":"minecraft:eye"}}
+		(netherite|black) eye = - {minecraft:trim={"material":"minecraft:netherite","pattern":"minecraft:eye"}}
+		(diamond|light blue) eye = - {minecraft:trim={"material":"minecraft:diamond","pattern":"minecraft:eye"}}
+		(gold|yellow) eye = - {minecraft:trim={"material":"minecraft:gold","pattern":"minecraft:eye"}}
+		(iron|grey|gray) eye = - {minecraft:trim={"material":"minecraft:iron","pattern":"minecraft:eye"}}
+		(copper|brown) eye = - {minecraft:trim={"material":"minecraft:copper","pattern":"minecraft:eye"}}
+
+		(emerald|green) spire = - {minecraft:trim={"material":"minecraft:emerald","pattern":"minecraft:spire"}}
+		(redstone|red) spire = - {minecraft:trim={"material":"minecraft:redstone","pattern":"minecraft:spire"}}
+		(lapis [lazuli]|blue) spire = - {minecraft:trim={"material":"minecraft:lapis","pattern":"minecraft:spire"}}
+		(amethyst [shard]|purple) spire = - {minecraft:trim={"material":"minecraft:amethyst","pattern":"minecraft:spire"}}
+		([nether] quartz|white) spire = - {minecraft:trim={"material":"minecraft:quartz","pattern":"minecraft:spire"}}
+		(netherite|black) spire = - {minecraft:trim={"material":"minecraft:netherite","pattern":"minecraft:spire"}}
+		(diamond|light blue) spire = - {minecraft:trim={"material":"minecraft:diamond","pattern":"minecraft:spire"}}
+		(gold|yellow) spire = - {minecraft:trim={"material":"minecraft:gold","pattern":"minecraft:spire"}}
+		(iron|grey|gray) spire = - {minecraft:trim={"material":"minecraft:iron","pattern":"minecraft:spire"}}
+		(copper|brown) spire = - {minecraft:trim={"material":"minecraft:copper","pattern":"minecraft:spire"}}
+armor trims:
```

Comment:
```suggestion
armor trims:
	minecraft version = 1.20 or newer
```

Reply:
Hang on I thought we were removing thesee?